README.md

```diff
@@ -42,6 +42,60 @@ Running the CLI will open a browser tab to `localhost:8000` to load the UI,
 and spin up a server listening on `localhost:4318` for OTLP http payloads and
 `localhost:4317` for OTLP grpc payloads.
 
+#### via Docker
```

Comment:
Documented how to use this.  Tried to not overexplain docker but also not leave the reader completely helpless.